### ISSUE TITLE
Controller extension break refined storage

### ISSUE THREAD

**Issue by peter-ahe** [Buuz135/FunctionalStorage#163]
I have a setup with multiple drawers connected to a controller that's connected to refined storage using refined storage's external storage block.

As soon as I link a controller extension to controller, I can't extract anything from refined storage, and its fluid terminal starts showing additional slots for water and lava. They're slowly appearing.

If I break the controller extension, the problem persists until I save/load the game.

Refined storage breaks again immediately after I link the extension.

![Screenshot 2023-02-19 at 10 20 37](https://user-images.githubusercontent.com/66904942/219939668-510f5a0a-fe4e-4fc9-b156-7f2df87d2c9a.png)

I'm using version 1.18.2-1.1.3 of function storage.

**Comment by Buuz135** [comment]
Do you connect RS to the Controller Extension too?

**Comment by peter-ahe** [comment]
> Do you connect RS to the Controller Extension too?

No. I just link it to the controller. The extension itself is not connected to RS.

**Comment by DoubleAWasTaken** [comment]
It's happening to me to on the Direwolf20 1.7.0 pack.  The extensions aren't actually extending the storage controllers reach either.

**Comment by zyoNoob** [comment]
does next-version tag means that it will be fixed with next functional_storage update? i.e the next release on git will resolve this?
